gateway/utils/routing/modify.go

```diff
@@ -82,9 +82,15 @@ func (r *Routing) modifyRequest(ctx context.Context, modules modulesInterface, r
 
 func (r *Routing) modifyResponse(res *http.Response, route *config.Route, token string, auth interface{}) error {
 	if res.Header.Get("Content-Type") == "application/json" {
-		data, err := ioutil.ReadAll(res.Body)
-		if err != nil {
-			return err
+		var data []byte
+		var err error
+		if route.Modify.Tmpl != "" {
+			data, err = ioutil.ReadAll(res.Body)
+			if err != nil {
+				return err
+			}
+		} else {
+			return nil
 		}
```

Comment:
Can we just return nil if `route.Modify.Tmpl == ""`? The code will be much cleaner that way